presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -375,41 +382,15 @@ public void setSchedulingPolicy(SchedulingPolicy policy)
             if (parent.isPresent() && parent.get().schedulingPolicy == QUERY_PRIORITY) {
                 checkArgument(policy == QUERY_PRIORITY, "Parent of %s uses query priority scheduling, so %s must also", id, id);
             }
-
-            // Switch to the appropriate queue implementation to implement the desired policy
-            UpdateablePriorityQueue<InternalResourceGroup> queue;
```

Comment:
Why was this refactor made after the code was already accepted? Please revert, and put in a separate commit if you want.